bindings/pyfiction/include/pyfiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -136,6 +134,22 @@ inline void operational_domain(pybind11::module& m)
 
         ;
 
+    m.def(
```

Comment:
@marcelwa I need your help with this function. When I use it, I get the error: ``TypeError: Unable to convert function return value to a Python type! The signature was
        (self: mnt.pyfiction.physically_valid_parameter_domain) -> phmap::parallel_flat_hash_map<fiction::parameter_point, unsigned long long, phmap::Hash<fiction::parameter_point>, phmap::EqualTo<fiction::parameter_point>, std::__1::allocator<std::__1::pair<fiction::parameter_point const, unsigned long long> >, 4ul, std::__1::mutex>``

Reply:
My guess is it is because Python doesn't have any `phmap` datatypes. Thus we cannot convert these to Python. We always have to work via getters here.

Reply:
okay, thank you.